config/serverless.yml

```diff
@@ -1,5 +1,5 @@
 newsfeed.enabled: false
-xpack.security.showNavLinks: false
+xpack.security.showNavLinks: true
```

Comment:
@sphilipse this config file is shared between all 3 projects I believe. did you try to overrid this in just `serverless.es.yml` first?

We may want to move this setting to the specific `serverless.___.yml` and remove it from this file, IF flipping this to true will effect security or observability

Reply:
No I forgot to set this back to false, had it to true to test some things out. Reverted now.